GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -967,7 +967,7 @@ private void InitializeComponent()
             this.toolStageAllItem.ImageTransparentColor = System.Drawing.Color.Magenta;
             this.toolStageAllItem.Name = "toolStageAllItem";
             this.toolStageAllItem.Size = new System.Drawing.Size(23, 23);
-            this.toolStageAllItem.Text = "Stage All";
```

Comment:
These should be removed here and made sure they are explicitly assigned in init or load
The text is translated twice now

Reply:
Agree. Good point. I can fix it together with the other change below.